src/main/java/generationExamples/ExamplesFromRandomGenerationImpl.java

```diff
@@ -0,0 +1,37 @@
+package generationExamples;
+
+import main.PositiveNegativeCounters;
+
+import java.util.List;
+
+public class ExamplesFromRandomGenerationImpl extends PositiveNegativeCounters implements EquationPattern {
+    private final static int SIZE_ARRAY = 5;
+    private final static int SIZE_ITEMS_OF_ARRAY = 3;
+
+    @Override
+    public Integer[][] generationExamples() {
+        Integer[][] listOfExamples = new Integer[SIZE_ARRAY][SIZE_ITEMS_OF_ARRAY];
```

Comment:
Let's try to use List<Integer[]> instead of Integer[][]

Reply:
done